clusterloader2/pkg/measurement/common/scheduling_throughput.go

```diff
@@ -32,153 +29,86 @@ import (
 
 const (
 	schedulingThroughputMeasurementName = "SchedulingThroughput"
-	defaultSchedulingThroughputInterval = 5 * time.Second
+
```

Comment:
It might be safer to instead of changing the existing measurement, create a new one next to the existing ones and keep two of them at the same time running for a bit of time.

In case of issues, the rollback is easy and you don't need to rollback everything - just disable it in tests.

Reply:
Actually that's exactly what I did before I merged them :D I moved it back to separate file.